crates/zkforge/src/runner.rs

```diff
@@ -201,6 +203,51 @@ impl<'a> ContractRunner<'a> {
         Ok(setup)
     }
 
+    /// Sets up the cheatcode address with an empty contract so `vm.func()` calls do not panic.
+    fn setup_cheatcode_address(&mut self) {
+        // https://github.com/matter-labs/era-test-node/blob/main/etc/system-contracts/contracts/EmptyContract.sol
+        let empty_contract_code =  "0000000101200190000000040000c13d0000000001000019000000110001042e0000008001000039000000400010043f0000000001000416000000000101004b0000000e0000c13d0000002001000039000001000010044300000120000004430000000501000041000000110001042e000000000100001900000012000104300000001000000432000000110001042e00000012000104300000000000000000000000020000000000000000000000000000004000000100000000000000000037118ec7e34bf260c2f7d3550e644dc0205a8f0a595d95265b1c50edc1c831ba";
+        let empty_contract_code_hash =
+            "010000071c769e7601af2a7d3b28a37d270c6dcc3daae65344627c5f83180be6";
+
+        self.executor
+            .backend
+            .insert_account_storage(
+                Address::from_slice(&ACCOUNT_CODE_STORAGE_ADDRESS.0),
+                U256::from_be_slice(
+                    &hex::decode(format!(
```

Comment:
we have `address_to_h256` in `zksync_utils`

Reply:
This is a different `alloy_primitive::Address`